Gemfile.lock

```diff
@@ -142,17 +142,17 @@ GEM
       warden (~> 1.2.3)
     diff-lcs (1.3)
     docile (1.1.5)
-    elasticsearch (6.3.0)
-      elasticsearch-api (= 6.3.0)
-      elasticsearch-transport (= 6.3.0)
-    elasticsearch-api (6.3.0)
+    elasticsearch (7.1.0)
+      elasticsearch-api (= 7.1.0)
```

Comment:
Need to roll this one back. We are getting ssl. Version errors on prod. Will pin elastic search to 6